### PR TITLE
Add Data Flow Diagram skeleton in the README

### PR DESCRIPTION
**Pull request by amanjeev**
There is a DFD in the README which is really helpful but services and
components are not detailed anywhere. This table needs to be populated
by the rest of the team who knows about the details of these components

Tickets:

- COGG-238

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,3 +147,13 @@ To deploy CanDIGv2, follow one of the available install guides in `docs/`:
...
 
 View additional Makefile options with `make help`.
+
+
+## Services and Components
+
+The follwing table lists the details from the Data Flow Diagram in the "Overview" section.
+
+| Service/Component Name                  | Source            | Notes                  |
+| ----------------------                  |----------------   |--------                |
```
Comment (shaikh-rashid) [inline README.md]: Looks good, could you start to add the additional components and tag who needs to provide the info for it?

**Review by shaikh-rashid — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (amanjeev) [reply README.md]: Yea I guess I would need to start somewhere. Perhaps @zxenia to begin the Katsu stuff. Thanks!

**Comment by amanjeev** [comment]
If you can merge this skeleton then everyone can add stuff to this table as they work on this repo and see fit

**Comment by amanjeev** [comment]
I had added the table to the README. Does this look good?

**Review by ljdursi — APPROVED** [review]
Looks good to me, but maybe call "CanDIG Server" something like "CanDIGv1 server (used for variant service)"?

**Pull request merged**